hotfixes.js

```diff
@@ -4,27 +4,42 @@ var hotfixes = {
   "Products.PloneHotfix20170117": {
     "required_for_plone": [
           ["3", "3.99.??"],
-          ["4", "4.99.??"],
-          ["5", "5.99.??"]
+          ["4", "4.3.11"],
```

Comment:
This constraint is inclusive (meaning: the hotfix is required for both versions in this list, and all between).

You're right: the version `5.0.7` is wrong here, it should be `5.0.6`. Corrected.